CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html

```diff
@@ -79,12 +79,16 @@ <h1>Plugin Reporter</h1>
             that have already been executed. The list of tags you can filter on may include:
         </p>
         <ul>
-            <li><em>Analytic</em> - Plugins which perform some analytic calculation based on information on the graph.</li>
-            <li><em>Import</em> - Plugins which add information to the graph, generally these will be found in the Data Access 
-                View.</li>
-            <li><em>General</em> - Plugins which don't fit into any specific category.</li>
+            <li><em>Create</em> - Plugins which create graphs, nodes, and transactions.</li>
+            <li><em>Modify</em> - Plugins that change the graph in some way.</li>
+            <li><em>Select</em> - Plugins which change what is selected on the graph.</li>
+            <li><em>Import</em> - Plugins which add data from other sources to the graph. Generally found in the Data Access View.</li>
+            <li><em>Clean</em> - Plugins which clean up the existing data on your graph. Found in the Data Access View.</li>
+            <li><em>Enrich</em> - Plugins which add more information to the existing nodes and transactions on your graph. Found in the Data Access View.</li>
+            <li><em>Extend</em> - Plugins which add more information with new nodes and transactions to your graph. Found in Data Access View.</li>
+            <li><em>Experimental</em> - Plugins which perform experimental features in Constellation.</li>
+            <li><em>Analytic</em> - Plugins which perform some analytic calculation based on information on the graph. Found in the Analytic View</li>
```

Comment:
Isn't there a UTILITY tag also?

Reply:
also Welcome ?

Reply:
@aldebaran30701 There are a few tags missing I've intentionally omitted. This list (according to the line leading in) is only a sample of the tags one could see. I could change this to the full list we expect if you think it is worthwhile but else I'm happy to leave it as is

Reply:
Ohh so the plugin reporter will not have access to all tags? Will plugin reports that are tagged as WELCOME appear when viewing all reports? Would seem off to me that if you can see them in the reporter, you should be able to filter with them (or omit the reports when viewing all) Interested in thoughts on that assumption

Reply:
You can see and filter on all the tags available, what I meant was I was only listing a few of the tags available on the help page

Reply:
For completeness I'd say its better to have every tag, I feel like if I was confused why there isn't all tags + explanations, a user could also be confused.

Reply:
@antares1470 I would prefer the full list please.

Reply:
Done